rpcserver.go

```diff
@@ -6441,6 +6442,81 @@ func (r *rpcServer) FundChannel(ctx context.Context,
 	}, nil
 }
 
+// EncodeCustomRecords allows RPC users to encode Taproot Asset channel related
+// data into the TLV format that is used in the custom records of the lnd
+// payment or other channel related RPCs. This RPC is completely stateless and
+// does not perform any checks on the data provided, other than pure format
+// validation.
+func (r *rpcServer) EncodeCustomRecords(_ context.Context,
```

Comment:
Do we still this as something that's temp? As in we'd just wrap the payment specific stuff in an entirely new RPC?

Reply:
I think so, yes. Or maybe not, given that with this it should be pretty easy to do the RPC calls in any language. I guess the question is: how much effort is it for the app devs to call 3 RPCs (buy/sell order, encode records, send payment) with the correct values vs. us threading all the payment and channel flags through some new RPC methods in tapd...